app/models/comment.rb

```diff
@@ -1,3 +1,5 @@
 class Comment < ApplicationRecord
   belongs_to :article
+  validates :commenter, presence: true
+  validates :body, presence: true
```

Comment:
This could be refactored to a single line, However comment functionality is a little bit off scope in this feature.